Rename bundles to addons (#100)
kyma-project/addons: docs/getting-started.md

2. Generate the access token. In your GitHub profile account, choose the **Settings** tab and click **Developer settings**. Then, go to **Personal access tokens** and **Generate new token**. Give your token a descriptive name. Select the `public_repo` scope for it. Click **Generate token** and copy your personal access token.

4. Sign up to [Travis CI](https://travis-ci.com/) with your GitHub account. Accept the Travis CI authorization.

5. Enable Travis CI. Click the **Activate** button and select your forked addons repository. Click the **Approve & Install** button to give Travis CI access to your forked `addon` repository.

6. Configure Travis CI. On the left sidebar, click **My Repositories** tab and choose your GitHub repository. Click the **More options** tab on the right and select **Settings**. Go to the **Environment Variables** section. Set **Name** as `GITHUB_TOKEN`. **Value** is equal to your GitHub personal access token. Click **Add**.

7. Trigger a build. Go to the **More options** tab and select **Trigger build**. Select the master branch and click **Trigger custom build**.

### Create your addons

> 1. [Create your addons](https://kyma-project.io/docs/master/components/helm-broker/#details-create-a-addon) and place them in the `addons/` directory.

Reviewer: ```suggestion
1. [Create your addons](https://kyma-project.io/docs/master/components/helm-broker/#details-create-addons) and place them in the `addons/` directory.
```

> 2. Update the [`index.yaml` file](https://kyma-project.io/docs/master/components/helm-broker/#details-create-a-addons-repository-index-yaml-file) which defines the available addons.

Reviewer: ```suggestion
2. Update the [`index.yaml` file](https://kyma-project.io/docs/master/components/helm-broker/#details-create-addons-repository-the-index-yaml-file) which defines the available addons.
```

3. Commit your changes and push them to the master branch.

4. Check the build status on [Travis CI](https://travis-ci.com/). To learn how to tag and release your addons, see [this](releases.md) document.

> 5. [Configure the Helm Broker](https://kyma-project.io/docs/master/components/helm-broker/#configuration-configuration) to fetch addons from your repository.

Reviewer: ```suggestion
5. Configure the Helm Broker to [fetch addons from your repository](https://kyma-project.io/docs/master/components/helm-broker/#details-fetch-addons-from-https-servers).
```
